### PR TITLE
misc: travis -> GH actions

### PR DESCRIPTION
**Pull request by lovesegfault**
Travis is being deprecated, GH actions is the new thing.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@ jobs:
           - stable
           - beta
           - nightly
-          - 1.29.0  # MSRV
```
Comment (est31) [inline .github/workflows/ci.yml]: Can you please bump it instead to whatever the MSRV is now?

Comment (lovesegfault) [reply .github/workflows/ci.yml]: I don't want to figure out what the MSRV is until I get the mentioned bindgen changes in, it'll be re-introduced later.

Also, this should only be merged after the refresh PR I have in parallel.

Comment (est31) [reply .github/workflows/ci.yml]: Alright.

**Comment by est31** [comment]
Travis CI should be disabled now.

**Comment by lovesegfault** [comment]
Alright, @est31 I can't get this through; GH is being weird; please force-merge it.

**Pull request merged**